airbyte-config/models/src/test/java/io/airbyte/config/helpers/S3LogsTest.java

```diff
@@ -22,7 +23,18 @@
 @Tag("logger-client")
 public class S3LogsTest {
 
-  private static final LogConfigs logConfigs = (new EnvConfigs()).getLogConfigs();
+  private static final String REGION_STRING = "us-west-2";
+  private static final Region REGION = Region.of(REGION_STRING);
+  private static final String BUCKET_NAME = "airbyte-kube-integration-logging-test";
+  private static final LogConfigs logConfigs = new EnvConfigs().getLogConfigs();
+
+  private S3Client s3Client;
+
+  @BeforeEach
+  void setup() {
```

Comment:
Why do you prefer relying on the external resource to keep state between test runs as opposed to just having the test guarantee the correct set up every time? Seems like it's breaking the pattern we try to use in almost all of our testing.

Reply:
It seems wasteful since we aren't really isolating each test run from the other (since we are always writing to the same location).

To properly isolate, we would also have to add a random prefix path and delete the old files. At that time I felt this is more trouble than it is worth, esp since the chance of something bad happening here is very small. (Someone would have to overwrite the same location)

I'm okay if you prefer the test set up doing this.